{% extends "base.html" %}

{% block content %}

<!-- Added this on 25 Oct 2022, 7:00pm (Fumin)
This is basically django default 500.html page so since debug = false, it will always throw server 500 error if anything
No need try/catch exception? Since it doesnt even catch at all, django will just throw this page-->

<html>
  <head>
    <meta http-equiv="refresh" content="2; url='http://127.0.0.1:8000/luna/'" />
  </head>
  <body><h1>The email is already taken, please try again</h1></body>
</html>


    {% endblock %}